packages/keystone/lib/ListTypes/hooks.js

```diff
@@ -119,6 +119,8 @@ class HookManager {
     if (fieldValidationErrors.length) {
       this._throwValidationFailure({ errors: fieldValidationErrors, operation, originalInput });
     }
+    // NOTE: addFieldValidationError should be awailable only for field level hooks
```

Comment:
awailable

Reply:
?

Reply:
He's trying to tell you to change `awailable` to `available` in the comment.